### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by JacopoBonta**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export class Challenge {
```
Comment (marmos91) [inline src/index.ts]: 🙈We use Allman style brackets (https://en.wikipedia.org/wiki/Indentation_style#Allman_style).
Put them on a new line

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    buzz(x: number):Promise<string | number> {
```
Comment (marmos91) [inline src/index.ts]: 🙈As above. Add also a space after punctation like `:`

File: src/index.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+        return new Promise((resolve, reject) => {
```
Comment (marmos91) [inline src/index.ts]: 🙈As above

File: src/index.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+            if (x % 3 === 0 && x % 5 === 0) return resolve('fizzbuzz')
```
Comment (marmos91) [inline src/index.ts]: That's ok but we prefer putting single line `if` blocks on a new line. Something like

```ts
if(condition)
    return true;
```

File: src/index.ts [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+            if (x % 3 === 0) return resolve('fizz')
+            if (x % 5 === 0) return resolve('buzz')
+            return resolve(x)
```
Comment (marmos91) [inline src/index.ts]: Missing semicolons

File: src/index.ts (hunk 3 above)
Comment (JacopoBonta) [reply src/index.ts]: Anche le arrow functions richiedono di andare a capo con le graffe?
Es:
```js
return new Promise((resolve, reject) =>
{
    // ...
}
```

**Pull request closed without merging**